ChromeExtension/js/openDialog.js

```diff
@@ -1,4 +1,4 @@
-chrome.extension.onMessage.addListener(function (msg, sender, sendResponse) {
+chrome.runtime.onMessage.addListener(function (msg, sender, sendResponse) {
```

Comment:
`chrome.extension.onMessage` is now unsupported, needs to be replaced:
https://developer.chrome.com/docs/extensions/develop/migrate/api-calls#replace-unsupported-apis